tests/unit/Common/Internal/Authentication/SharedAccessSignatureHelperTest.php

```diff
@@ -233,4 +233,377 @@ public function testGenerateAccountSharedAccessSignatureToken()
             $this->assertEquals($testCase[8], urlencode($actualSignature));
         }
     }
+
+    public function testGenerateServiceSharedAccessSignatureToken()
+    {
+        // Setup
+        $accountName = "phptests";
+        $accountKey = "WaZvixrkMok53QDmj8Tc99+BV6vO9cCOJNsdm+wD9QVEScwl8c1eYPcQ182ndNFqxX1+SEKs18SmOxh8OpzIUg==";
```

Comment:
Assume that this is a fake key. There is already one [here](https://github.com/Azure/azure-storage-php/blob/master/tests/framework/TestResources.php#L54), Please use this one.